README.md

```diff
@@ -59,54 +69,114 @@ Some components or modules in forte may require some [extra requirements](https:
-installing forte spacy wrapper)
+## Quick Start Guide
+Here we provide an example for user to get started with Forte easier. Consider a case we want to get words in the sentence and also extract entities from the sentence. We can write two processors for the two separate tasks.
 
+First, we imports all required libraries.
 ```python
 from forte import Pipeline
+from forte.processors.base import PackProcessor
+from forte.data.data_pack import DataPack
+from dataclasses import dataclass
+from forte.data.ontology.top import Generics
+from typing import Optional, Dict, Any
 from forte.data.readers import TerminalReader
 from fortex.spacy import SpacyProcessor
+import re
+```
+In Forte, we use data entry with different entry types to represent data.
+The following class is an example of data entry storing word texts in the class variable `value`.
+New User doesn't need to pay too much attention about it and can just use it as it is.
 
-for pack in Pipeline().set_reader(
-        TerminalReader()
-).add(
-    SpacyProcessor(), {"processors": ["sentence", "ner"]}
-).initialize().process_dataset():
+```python
+@dataclass
+class Word(Generics):
+    """A dummy generic type for words."""
+    value: Optional[str] = None
+    def __init__(self, pack, value):
+        super().__init__(pack)
+        self.value = value
+```
+Next, we can write a simple customized processor for the first task, split sentences into words.
+There are two steps for the sentence processing.
+First, we need to strip all punctuation as split words should not contain them.
+Second, we need to split stripped sentences into words and write words into the data pack.
+```python
+class WordSplitPackProcessor(PackProcessor):
+    """A processor that removes punctuation in sentences in the data pack
+    and split sentences into words and write words into the data pack.
+    """
+    def __init__(self):
+        super().__init__()
+
+    def initialize(self, resources, configs):
+        super().initialize(resources, configs)
+
+    def _process(self, input_pack: DataPack):
+        # write a data type Word into the data pack
+        # since in the first processor, it writes sentence into the
+        # data pack, we can get sentences by DataPack.get() which
+        # generate data entries of parameter type.
+        for sentence in input_pack.get("ft.onto.base_ontology.Sentence"):
+            # first step: strip all punctuation
+            words = re.sub(r'[^\w\s]','',sentence.text).split(" ")
+            # second step: split stripped sentences into words and write
+            # words into the data pack
+            for w in words:
+                # we pass the input pack and word strings
+                # into `Word` class, and the `Word.value` store
+                # `input_pack` will internally add `Word` entries into `input_pack`'s internal storage
+                # while `Word` initializes itself.
+                Word(pack=input_pack, value=w)
```

Comment:
Use something more useful here, `value` doesn't help too much.  Just use https://github.com/asyml/forte/blob/master/ft/onto/base_ontology.py#L61.

Another reason is this `Word` class is for test purposes, *never* use it for any production thing. `Word` is not even inheriting an Entry class. So this will give a lot of wrong information.

If you want to assign some useful values, how about just using NLTK pos tagger: https://www.nltk.org/book/ch05.html

Reply:
OK. I didn't consider the difference between `Generics` and `Entry`. I am found that there is a NLTK tagger in forte wrapper. Maybe I can import `fortex.nltk.NLTKWordTokenizer` and simplify the implementation of  `fortex.nltk.NLTKPOSTagger` and put the code here?

Reply:
1. If you use `Word`, you will want to use the `begin` and `end`, not use a meaningless `value` here. This example will just give people the wrong impression
2. Yes, make it really simple, something you expect one can understand the second he read the code. You can also consider use this whitepsace tokenize too. https://github.com/asyml/forte/blob/master/forte/processors/misc/simple_processors.py#L43